lib/pages/new_private_chat/new_private_chat.dart

```diff
@@ -42,7 +46,18 @@ class NewPrivateChatController extends State<NewPrivateChat>
   }
 
   void goToNewGroupChat() {
-    context.push('/rooms/newprivatechat/newgroup');
+    if (_responsiveUtil.isDesktop(context) ||
+        _responsiveUtil.isTablet(context)) {
```

Comment:
you use this condition a lot please create a isDesktopOrTablet for responsiveUtils

Reply:
good idea.